src/nuxeo-cloud/day-to-day.md

```diff
@@ -0,0 +1,43 @@
+---
+title: 'Nuxeo Cloud Day-to-day'
+review:
+    comment: ''
+    date: '03-19-2020'
+    status: ok
+labels:
+    - nuxeo-cloud
+tree_item_index: 300
+---
+
+<div class="row" data-equalizer data-equalize-on="medium">
+<div class="column medium-4">
+{{#> panel type='secondary' match_height='true'}}
+### Deployment
+
+The Nuxeo Cloud Operations (NCO) team will deploy all changes and updates to the pre-prod and prod environments.
+<br/>
+
+[More&nbsp;<i class="fa fa-long-arrow-right" aria-hidden="true"></i>]({{page page='deployment'}})
+{{/panel}}
+</div>
+<div class="column medium-4">
+{{#> panel type='secondary' match_height='true'}}
+### Security
+
+All environments are enabled with out of the box application and infrastructure level security monitors.
+<br/>
+
+[More&nbsp;<i class="fa fa-long-arrow-right" aria-hidden="true"></i>]({{page page='security'}})
+{{/panel}}
+</div>
+<div class="column medium-4">
+{{#> panel type='secondary' match_height='true'}}
+### Support and Maintenance
+
+All environments are proactively monitored 24x7 and supported by the Nuxeo Cloud team.
```

Comment:
All environments are enabled with 24x7 monitoring and supported by the Nuxeo Cloud team